benchmarks/benchmark_throughput.py

```diff
@@ -422,6 +481,11 @@ def main(args: argparse.Namespace):
                         action='store_true',
                         default=False,
                         help="Disable decoupled async engine frontend.")
+    # LoRA
+    parser.add_argument("--lora-path",
+                        type=str,
+                        default='yard1/llama-2-7b-sql-lora-test')
```

Comment:
Not recommended to use `default` , as LoRA is strongly coupled bound to the model. We also need to add  `help`  to describe this arg

Reply:
You are correct. I have removed the default :+1:  Thanks !